CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing to AI Validation
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways
+to help and details about how this project handles them. Please make sure to read the relevant section before making your
+contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The
+community looks forward to your contributions. 🎉
+
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [I Have a Question](#i-have-a-question)
+- [I Want To Contribute](#i-want-to-contribute)
+- [Reporting Bugs](#reporting-bugs)
+- [Suggesting Enhancements](#suggesting-enhancements)
+- [Styleguides](#styleguides)
+- [Commit Messages](#commit-messages)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[Code of Conduct](blob/main/CODE_OF_CONDUCT.md).
```

Comment:
@berrydenhartog on the published version of the page this link will not work. What do you think of the suggested change? If you agree, we should also update the other links in this document.

```suggestion
[Code of Conduct](https://github.com/MinBZK/ai-validation/blob/main/CODE_OF_CONDUCT.md).
```

Reply:
Resolved